test/tests/functional/pbs_test_tpp.py

```diff
@@ -354,10 +354,12 @@ def test_comm_with_vnode_insertion(self):
         """
         Test for verifying vnode insertion when using TPP
         """
+        vn = self.mom.shortname
         if not self.mom.is_cpuset_mom():
             a = {'resources_available.ncpus': 1}
-            self.server.create_vnodes('vn', a, 2, self.mom)
-            vnode_val = "vnode=vn[0]:ncpus=1+vnode=vn[1]:ncpus=1"
+            self.mom.create_vnodes(a, 2)
+            vnode_val = "vnode=" + vn + \
+                "[0]:ncpus=1+vnode=" + vn + "[1]:ncpus=1"
```

Comment:
Consider using string formatting.  If you do that, you might be able to put the string on the first line and the (vn, vn) on the second.